recipe/recipe.yaml

```
@@ -1,27 +1,30 @@
-{% set name = "sbi" %}
-{% set version = "0.24.0" %}
-{% set python_min = "3.10" %}
+schema_version: 1
+
+context:
+  name: sbi
+  version: 0.24.0
+  python_min: 3.10
 
 package:
-  name: {{ name|lower }}
-  version: {{ version }}
+  name: ${{ name|lower }}
+  version: ${{ version }}
 
 source:
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/sbi-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/sbi-${{ version }}.tar.gz
   sha256: 86a436bf6ccdfeb04085085e8db07aed2e86dd32c3dc88c5f2c0385fd7c46396
 
 build:
+  number: 1
   noarch: python
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  number: 0
+  script: ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
-    - python {{ python_min }}
+    - python 3.10.*
```

```suggestion
    - python ${{ python_min }}.*
```

Fixup given bug in https://github.com/conda-incubator/conda-recipe-manager/issues/307